.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Not sure if this was an intended addition. Ideally, we should use nvmrc from OSD.